thirdparty/README.md

```diff
@@ -359,6 +359,11 @@ Files extracted from upstream source:
 - all .cpp and .h files apart from `main.cpp`
 - LICENSE.TXT
 
+## recast
+
+- Upstream: https://github.com/recastnavigation/recastnavigation
+- version: git commit ef3ea40f - 2016-02-06
```

Comment:
It's actually 2017-02-06 :)

Could you also document the license and copyright in the `COPYRIGHT.txt` file?